health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/AppConfigurationWrapper.js

```
@@ -536,6 +536,8 @@ function AppConfigurationWrapper({ screenConfig, localeModule, pageTag }) {
     // setShowToast({ key: "success", label: "LOCALISATION_SUCCESS" });
   };
 
+  const currentPage = parseInt(pageTag.split(" ")[1]);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)